src/test/java/com/amazonaws/encryptionsdk/CMMHandlerTest.java

```diff
@@ -3,139 +3,158 @@
 
 package com.amazonaws.encryptionsdk;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
 import com.amazonaws.encryptionsdk.model.DecryptionMaterials;
 import com.amazonaws.encryptionsdk.model.DecryptionMaterialsHandler;
 import com.amazonaws.encryptionsdk.model.DecryptionMaterialsRequest;
 import com.amazonaws.encryptionsdk.model.KeyBlob;
-import org.junit.Test;
-
 import java.security.PublicKey;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import org.junit.Test;
```

Comment:
TBH, I prefer that we keep these. But I don't know how to make that work, or want to block on it.

Reply:
We are keeping these! They just got moved to the top of the file in checkstyle reformatting